Feature/schedule

jillwc07 opened the pull request.

jillwc07 committed:
Add template skeleton layout for conference schedule

jillwc07 committed:
Update top paragraph, title, content, dates and location of conference

jillwc07 committed:
Update top paragraph, title, content, dates and location of conference

jillwc07 committed:
Content update to wrtie, speak, code, self-care day tabs. Add custom class for styling

jillwc07 committed:
Content update to wrtie, speak, code, self-care day tabs. Add custom class for styling

jillwc07 committed:
Assign custom styling to schedule top paragraph to be visible in mobile view. Modify spacing and padding to text and  bullet points.

jillwc07 committed:
Assign custom styling to schedule top paragraph to be visible in mobile view. Modify spacing and padding to text and  bullet points.

jillwc07 committed:
Update schedule link in mobile nav.

jillwc07 committed:
Update schedule link in mobile nav.

jillwc07 committed:
Add media queries to improve mobile responsive views.

jillwc07 committed:
Add media queries to improve mobile responsive views.

jillwc07 committed:
Add new class and modify css to fix active tab bar from covering self care and career day text in mobile view.

jillwc07 committed:
Add new class and modify css to fix active tab bar from covering self care and career day text in mobile view.

jillwc07 committed:
Add missing } to media query.

jillwc07 committed:
Add missing } to media query.

jillwc07 committed:
Update schedule tab, spelling fix, mobile responsive fix.

jillwc07 committed:
Add template skeleton layout for conference schedule

jillwc07 committed:
Add padding and remove duplicate schedule tab

jillwc07 committed:
Merge branch 'master' into feature/schedule

jillwc07 committed:
fix merge conflicts

jarmstrng commented on assets/css/custom.css:
```diff
@@ -307,6 +307,105 @@ section.speakers .qcTabTitle .section-title {
+@media only screen and (max-width: 360px){
+	.wsc-schedule-tabs > ul li {
+	  width: 65px !important;
+	}
```
there are a lot of `!important`s, did you have trouble overriding the default styles?

jillwc07 committed:
Update id to class for border to appear under schedule section.

jillwc07 committed:
Change schedule tab color to coral.

jillwc07 committed:
Add schedule section for write/speak/code day.

jillwc07 committed:
reduce commits

Merge branch 'feature/schedule' of https://github.com/WriteSpeakCode/2017.writespeakcode.com into feature/schedule
